decorator.rb

```diff
@@ -0,0 +1,22 @@
+require './nameable'
+
+class Decorator < Nameable
```

Comment:
- [ ] This class should contain a method correct_name, kindly consider implementing it
```
    def correct_name
      @nameable.correct_name
    end
```